### PR TITLE
Migrate content out of FAQ page

### PR DESCRIPTION
**Pull request by s-makin**
```
Migrate content out of FAQ page (SD-1187)

Extracted several short topics out of the FAQ and
created individual "how-to" pages. Now what's left in 
the FAQ is mostly troubleshooting content.
```

### CODE REVIEW

File: doc/rtd/reference/faq.rst [hunk 1]
```diff
@@ -15,176 +15,20 @@ Having trouble? We would like to help!
...
-
 Why did ``cloud-init`` never complete?
 ======================================
 
 To check if ``cloud-init`` is running still, run:
 
 .. code-block:: shell-session
 
-        $ cloud-init status
+   cloud-init status
```
[the remark this answers is not in the document]
Comment (s-makin) [reply doc/rtd/reference/faq.rst:25]: Yeah, I'm gradually removing them everywhere (whenever possible).

**Comment by s-makin** [comment]
> FYI @s-makin. For the jira item we prefer not to use markdown format as you have it now:
> 
> ```
> Extracted several short topics out of the FAQ and
> created individual "how-to" pages. Now what's left in 
> the FAQ is mostly troubleshooting content.
> 
> [SD-1187](https://warthogs.atlassian.net/browse/SD-1187)
> ```
> 
> more like this would be better:
> 
> ```
> Migrate content out of FAQ page (SD-1187)
> 
> Extracted several short topics out of the FAQ and
> created individual "how-to" pages. Now what's left in 
> the FAQ is mostly troubleshooting content.
> ```

Thanks for the headsup, I changed it now

File: doc/rtd/reference/faq.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (igalic) [reply doc/rtd/reference/faq.rst:25]: a $ to me implies running a command as regular user. if i do and inevitably run into trouble, my first thought will be that something is wrong with my setup

File: doc/rtd/howto/debug_user_data.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Static user data validation
+---------------------------
+
+To verify your cloud config is valid YAML you can use `validate-yaml.py`_.
+
+To ensure the keys and values in your user data are correct, you can run:
+
+.. code-block:: shell-session
+
+    cloud-init schema --system --annotate
```
Comment (blackboxsw) [inline doc/rtd/howto/debug_user_data.rst:18]: This `--system` requires root user to normal user can't do this. let's provide sudo here
```suggestion
    sudo cloud-init schema --system --annotate
```

File: doc/rtd/howto/debug_user_data.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Or, to test YAML in a file:
+
+.. code-block:: shell-session
+
+    cloud-init schema -c test.yml --annotate
```
Comment (blackboxsw) [inline doc/rtd/howto/debug_user_data.rst:24]: No sudo needed here because we are passing a flat file we presumably own.

**Review by blackboxsw — COMMENTED** [review]
Minor nit and this can land. Thanks @s-makin

**Pull request merged**